common/cache/domainCache.go

```diff
@@ -952,3 +958,15 @@ func (entry *DomainCacheEntry) IsSampledForLongerRetention(
 	}
 	return false
 }
+
+// HoursToAddToRetentionDuration return number of hours to add to jittered wf deletion time
+func (entry *DomainCacheEntry) HoursToAddToRetentionDuration() time.Duration {
+	if strHours, ok := entry.info.Data[JitteredWorkflowDeletionHoursKey]; ok {
```

Comment:
Just a general wondering.
We usually use dynamicconfig with some default value and overrides per domain if needed. While this would work, it does not seem to provide a configurable default for all domains.

Reply:
It seems that the other retention configuration is done in the same way. IMHO we should either move it all into dynamic config somehow or not at all. It seems random to have 1 field be dynamic config then another in the info map since they're both related to retention duration calculation. Wdyt?

Reply:
IMO the issue with this approach, is there is no convenient way to apply a default for the whole cluster, which is what we want to do. With maybe some exceptions for a few domains if needed.

Reply:
Yes I can change the default to be jittered instead and use the flag to not jitter or a different jitter config 👍